orchestration/hca_orchestration/solids/load_hca/utilities.py

```diff
@@ -25,16 +26,22 @@ def validate_and_notify(
         data_repo_client=context.resources.data_repo_client,
         snapshot=False
     ).check_for_all()
-    if check_result.has_problems():
-        raise Failure("Dataset failed validation")
 
-    kvs = {
-        "Staging area": context.run_config["solids"]["pre_process_metadata"]["config"]["input_prefix"],
-        "Target Dataset": context.resources.target_hca_dataset.dataset_name,
-        "Jade Project": context.resources.target_hca_dataset.project_id,
-        "Dagit link": f'<{context.resources.dagit_config.run_url(context.run_id)}|View in Dagit>'
-    }
-    context.resources.slack.send_message(blocks=key_value_slack_blocks("HCA Completed Import", key_values=kvs))
+    if check_result.has_problems():
+        message = construct_validation_message(
+            check_result,
+            target_hca_dataset.dataset_name,
+            short_run_id(context.run_id)
+        )
+        context.resources.slack.send_message(message)
+    else:
+        kvs = {
+            "Staging area": context.run_config["solids"]["pre_process_metadata"]["config"]["input_prefix"],
+            "Target Dataset": context.resources.target_hca_dataset.dataset_name,
+            "Jade Project": context.resources.target_hca_dataset.project_id,
+            "Dagit link": f'<{context.resources.dagit_config.run_url(context.run_id)}|View in Dagit>'
+        }
+        context.resources.slack.send_message(blocks=key_value_slack_blocks("HCA Completed Import", key_values=kvs))
```

Comment:
Is "HCA Completed Import" correct here? It was already this way before your changes but just wondering because I thought this step was just validating and notifying.